build.gradle

```diff
@@ -205,6 +206,7 @@ subprojects {
         consoleOutput = true
         ruleSetFiles = files(rootProject.file('config/pmd/pmd.xml'))
         ruleSets = []
+        incrementalAnalysis = true
```

Comment:
I believe this might actually slow things down for travis.
@ssoloff once commented on the warning, but I don't recall exact details

Reply:
IIRC his comment was that there was not a good way to turn off the warning, that incremental was not supported on gradle.

> I believe this might actually slow things down for travis.

Why do you suspect that? 

Seems like it is simply caching results: https://pmd.github.io/latest/pmd_userdocs_incremental_analysis.html
I would be a bit surprised if that were noticeably slow. 

Failing turning on incremental, it would be good to find a way to suppress the output warnings. It was a significant problem finding errors in a lot of 'noisy' log messages.

Reply:
This was the comment/message about the warning from @ssoloff  that I was recalling (see the additional comments section): https://github.com/triplea-game/triplea/pull/4637

Reply:
My comment was that the warning wasn't suppressable at the time PMD was introduced into TripleA (see #4637).  It looks like the `incrementalAnalysis` property was just recently introduced in Gradle 5.6, so I have no idea how it might or might not affect build performance.  The only thing I'd recommend is to verify that, with this option enabled, a known PMD violation still triggers a build failure.

Reply:
Ah ok, misconception on my side, I had remembered the conversation a little bit different.